test/go.test.ts

```diff
@@ -220,7 +220,7 @@ encountered.
 			[new vscode.Position(22, 5), 'main func()', '\n'],
```

Comment:
this seems to fail with an extra line
`      AssertionError [ERR_ASSERTION]: '\n```go\nmain func()\n```\n' == '\n```go\nmain func()\n```\n\n'
      + expected - actual
       
       ```go
       main func()
       ```
      +
     `